README.md

```diff
@@ -162,7 +162,7 @@ Create remote data provider by calling `CompleterService.remote`.
 |imageField|string|Name of the field to use as image url for the list item.|
 |urlFormater|(term: string) => string|Function that get's the searchterm and returns the search url before each search.|
 |dataField|string|The field in the response that includes the data.|
-|headers|Headers (@angular/http)|HTTP request headers that should be sent with the search request.
```

Comment:
Agree with you it makes sense that if `requestOptions` are used we ignore `headers`